### ISSUE TITLE
Jonah issue (BCS)

### ISSUE THREAD

**Issue by BincyJ** [unfoldingWord/tc-create-app#460]
Unable to open the Jonah tN. The app crashes each time file is clicked.

https://git.door43.org/translationCore-Create-BCS/hi_tn/src/branch/master/manifest.yaml

**Comment by birchamp** [comment]
@mandolyte please add what you find out about this.

**Comment by BincyJ** [comment]
@mandolyte have you had a chance to take a look at it?

**Comment by mandolyte** [comment]
@BincyJ No, but I just tried and snapped the log message. Appears to be related to this function `getColumnsFilterOptions`, with which I am not familiar.

I'll take a look tomorrow. In the meantime, @ancientTexts-net @RoyalSix is this a familiar function? Probably a data issue with the TSV file, but it obviously just shouldn't crash!


![image](https://user-images.githubusercontent.com/1289437/97490210-ea9cb900-1936-11eb-8066-e9da95837c0f.png)

**Comment by mandolyte** [comment]
@BincyJ I notice that the manifest is marked invalid. The book of Jonah is missing, as is Ruth. These are the only two OT books in this repo. The manifest is causing the CV app to hang... so I can't run the validation on Jonah.

Who can we ask to update the manifest?

**Comment by birchamp** [comment]
@richmahn can you add the OT to the https://git.door43.org/translationCore-Create-BCS/hi_tn/src/branch/master/manifest.yaml ?

**Comment by richmahn** [comment]
@mandolyte Manifest being invalid just means it doesn't meet the Catalog Next manifest.yaml schema, as we wanted to make sure issued and modified fields had valid dates, as well as contributors be an empty array, not null. Fixed that and added Jonah & Ruth.

**Comment by mandolyte** [comment]
@richmahn Thanks!
@BincyJ the CV app now works on Jonah at the BCS Hindi repo.

Was this a CV app issue? Perhaps it should be moved there?

OK to close??

**Comment by birchamp** [comment]
@richmahn tC Create crashed when opening Jonah

**Comment by mandolyte** [comment]
Still crashes... the error in the console indicates an issue with column filters. As I recall, there are several current issues being worked in this space. Perhaps it can be tested in one of those previews.

Here is the console error:
![image](https://user-images.githubusercontent.com/1289437/98043734-ba03c600-1df3-11eb-8ab7-01d7a66dc5b7.png)

**Comment by birchamp** [comment]
@mandolyte production tC Create crashes?

**Comment by mandolyte** [comment]
@birchamp Yes, with production - I mention this to point out that the manifest fix did not help.

**Comment by birchamp** [comment]
> ...The manifest is causing the CV app to hang... so I can't run the validation on Jonah.
> 

@mandolyte so now that the manifest is updated can the CV app check it?

**Comment by mandolyte** [comment]
Yes! there are 314 tN errors, with 4 in the "red" level.


Cecil

**Comment by birchamp** [comment]
Can you do a screencast of running that check and jumping to a specific error in DCS and we'll send it to Vipin with the link to the CV app.

**Comment by mandolyte** [comment]
I don't know if I have any way to make a screencast. Could annotate some
screenshots and put them into a PDF... would that work?

**Comment by mandolyte** [comment]
@ancientTexts-net @birchamp Jonah in Hindi at BCS still crashes on the latest develop deployment.
https://develop--tc-create-app.netlify.app/
